src/commands/pickFuncProcess.ts

```diff
@@ -15,12 +15,50 @@ import { localize } from '../localize';
 import { delay } from '../utils/delay';
 import { requestUtils } from '../utils/requestUtils';
 import { taskUtils } from '../utils/taskUtils';
-import { ProcessDataFlag, getWindowsProcessTree, type IProcessInfo, type IWindowsProcessTree } from '../utils/windowsProcessTree';
+import { getWindowsProcessTree, ProcessDataFlag, type IProcessInfo, type IWindowsProcessTree } from '../utils/windowsProcessTree';
 import { getWorkspaceSetting } from '../vsCodeConfig/settings';
 
 const funcTaskReadyEmitter = new vscode.EventEmitter<vscode.WorkspaceFolder>();
 export const onDotnetFuncTaskReady = funcTaskReadyEmitter.event;
 
+export async function startFuncProcessFromApi(
+    workspaceFolder: vscode.WorkspaceFolder,
+    buildPath: string,
+    args?: string[]
+): Promise<{ processId: string; success: boolean; error: string }> {
+    const result = {
+        processId: '',
+        success: false,
+        error: ''
+    };
+
+    let funcHostStartCmd: string = 'func host start';
+    if (args) {
+        funcHostStartCmd += ` ${args.join(' ')}`;
+    }
+
+    await callWithTelemetryAndErrorHandling('azureFunctions.api.startFuncProcess', async (context: IActionContext) => {
+        try {
+            await waitForPrevFuncTaskToStop(workspaceFolder);
```

Comment:
Would it be ok to track FuncTasks by `buildPath` instead of workspaceFolder? This is to support multiple AzureFunction projects in an open folder. E.g. Solution with multiple Azure Functions projects.

Reply:
Addressed in https://github.com/microsoft/vscode-azurefunctions/pull/4309